File: server/src/main/java/com/lisowski/server/models/RideDetails.java
package com.lisowski.server.models;

import lombok.Data;
import org.hibernate.annotations.UpdateTimestamp;

import javax.persistence.*;
import java.time.Instant;


@Data
@Entity
@Table(name = "ride_deteils")
public class RideDetails {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Instant timeStart;
    private Instant timeArriveToUser;
    private Instant timeEnd;
    private String startPoint;
    private String waypoint;
    private String endPoint;
    @Column(length = 1500)
    private String userPolyline;
    @Column(length = 1500)
    private String driverPolyline;
    private Long userDistance;
    private Long driverDistance;

    private float price;

    private Integer rating;

    @OneToOne(mappedBy = "rideDetails")
    private Ride ride;

}

File: DriverApp/app/src/main/java/com/lisowski/driverapp/models/RideDetails.kt
package com.lisowski.driverapp.models

import java.time.Instant

data class RideDetails(
    val idRide: Long,
    val idUser: Long,
    val idDriver: Long,
    val userDistance: Long,
    val driverDistance: Long,
    val timeStart: String,
    val arrivalTime: String,
    val endTime: String,
    val userPolyline: String,
    val driverPolyline: String,
    val userLocation: String,
    val userDestination: String,
    val rating: Int,
    val price: Float
) {
}